app/src/main/java/com/android/streetworkapp/model/park/ParkViewModel.kt

```diff
@@ -40,6 +40,10 @@ open class ParkViewModel(
     _currentPark.value = park
   }
 
```

Comment:
Improvement: since `setCurrentPark` has a doc could you also add a small function description ?